### ISSUE TITLE
Autoprefixer fails on combination of @supports and grid-template-columns

### ISSUE THREAD

**Issue by smashercosmo** [postcss/autoprefixer#1244]
The following snippet 
```css
@supports (grid-template-columns: 1fr) {
  .columns {
    grid-template-columns: 1fr;
  }
}
```
results in error
```
TypeError: Cannot read property 'grid' of undefined
```

**Comment by ai** [comment]
@smashercosmo sorry, I missed this issue.

This code works without any problem:

```js
let autoprefixer = require('.')
let postcss = require('postcss')

let css = `
@supports (grid-template-columns: 1fr) {
  .columns {
    grid-template-columns: 1fr;
  }
}`

console.log(postcss([autoprefixer({ grid: 'autoplace' })]).process(css).css)
```

Maybe the bug was fixed? Can you check it again? If the issue is still actual, can you give me full stack trace?

**Comment by smashercosmo** [comment]
If you paste this code into https://autoprefixer.github.io/ you'll get this error (but it's possible that this service uses the old version of autoprefxer)

**Comment by smashercosmo** [comment]
Yes, it uses autoprefixer@9.6.1

**Comment by ai** [comment]
I got the correct output in the online tool too. What browsers do you have there?

**Comment by smashercosmo** [comment]
You will this error if you set ```last 4 version``` in the filter field

**Comment by ai** [comment]
Fixed cc827ab

**Comment by smashercosmo** [comment]
:+1::+1::+1: thx

**Comment by ai** [comment]
Released in 9.7
